### ISSUE TITLE
a11y console errors

### ISSUE THREAD

**Issue by danielsogl** [reactstrap/reactstrap#1670]
- components: `Dropdown`
- reactstrap version `8.1.0`
- import method `umd` 
- react version `latest`
- bootstrap version `latest`

### What is happening?

The new property a11y is throwing errors because of the boolean value `true` that is set now by default.

### What should be happening?

The prop should be set as described by react

### Steps to reproduce issue

1. use a drop down or any other component with the new a11y prop
2. run the app and open the console

### Error message in console

```
Warning: Received `true` for a non-boolean attribute `a11y`.

If you want to write it to the DOM, pass a string instead: a11y="true" or a11y={value.toString()}.
```

**Comment by rhdtl78** [comment]
I think it comes from [here](https://github.com/reactstrap/reactstrap/pull/1600)

**Comment by TheSharpieOne** [comment]
fixed in #1673
